README.md

```diff
@@ -117,8 +117,6 @@ the 2023 assessment model.
 | Condominium Unit Full Baths                                             | Characteristic | numeric   | X                     |
 | Condominium Building Is Mixed Use                                       | Characteristic | logical   | X                     |
 | Condominium % Ownership                                                 | Meta           | numeric   | X                     |
-| Condominium Building Strata 1                                           | Meta           | character | X                     |
-| Condominium Building Strata 2                                           | Meta           | character | X                     |
```

Comment:
**issue:** Strata are definitely still features. Why are these booted?

Reply:
Good question.

Reply:
So, we removed strata from the vars dict [here](https://github.com/ccao-data/ccao/commit/8b4ed8409a580d1aae54cf00e44bb759bc3fa483#diff-8afb1d683273af52aa9bfbe3d24043712c275e1a0aea3b0f34ea5e66d133d1a4L486). You okay with me adding it back?

Reply:
We should also wrap the current ccao PR before we merge this PR.

Reply:
I think I'm a little confused. In the res readme there's also an [`inner_join()`](https://github.com/ccao-data/model-res-avm/blob/5a33b6e043325936dc4b83a77d1ceffebe7b8574/README.Rmd#L311) against `ccao::vars_dict` for feature names which seems like it forces all feature names to be in that data?

Reply:
That might actually be a mistake. I _think_ that join should be reversed, with `param_tbl` on the left side and `vars_dict` left joined. Calling @jeancochrane to confirm.

Reply:
@dfsnow Yup, I think that is indeed a bug. Looks like we had a conversation about [the relevant code block](https://github.com/ccao-data/model-res-avm/pull/18#discussion_r1346330141) where you suggested a different join method and I copy/pasted your suggestion such that I removed the correct order of the join (which still should have been a left join instead of an inner join). My bad for not thinking it through at the time!

Reply:
Alright, in that case I'll work on wrapping up this PR then we can actually revert the CCAO package back a few merges I believe.